### ISSUE TITLE
Tags show commit should open commit same as blame

### ISSUE THREAD

**Issue by yyyc514** [divmain/GitSavvy#106]
Any reason the behavior here is not standardized?  One opens a new tab, the other uses an output panel?  I could submit a PR to standardize this.  Can we agree on a single behavior?  Or a configuration option to choose behavior?

**Comment by divmain** [comment]
The reason was that I didn't catch it in review :)  @p3lim, was there a reason you preferred the panel over a scratch view?  I prefer the scratch view because 1) it doesn't disappear after closing/re-opening sublime, and 2) it is easier to review long diffs.  However, I'm not averse to making this configurable if there's a good reason.

**Comment by yyyc514** [comment]
Oh I'd prefer scratch also.

**Comment by p3lim** [comment]
Personally, I prefer the panel, simply because it doesn't take me out of the original view I was in, it just overlays with temporary information (commits/diffs), which could become especially useful in the graph view.

I can agree on it not being the default method of showing it, but I would like to see an option for it somewhere down the road.

**Comment by divmain** [comment]
Fair enough. And there's no need to change the behavior without making it configurable as part of the same effort.

**Comment by yyyc514** [comment]
What is needed here?  A global setting for all "open related view in" or is this something where we need a hash and someone should be able to choose how they want content opened depending on the source of the open request?

**Comment by p3lim** [comment]
A global setting, but this is probably put on hold until @divmain ends his merge to the new interface model.

**Comment by divmain** [comment]
Thanks for waiting :)  Status view is ported, and tag view is up next.  Once that is done, this should have the green light.

**Comment by divmain** [comment]
This was completed as part of 7dc5438b773cf8fde9b4d01a76c6af1c55bcc7d6.
